src/Honeycomb/Api/Types/HoneyValue.hs

```diff
@@ -69,3 +70,13 @@ instance JSON.ToJSON HoneyValue where
   toJSON (HoneyString s) = JSON.String s
   toJSON (HoneyNumber n) = JSON.Number n
   toJSON (HoneyBool b) = JSON.Bool b
+
+instance JSON.FromJSON HoneyValue where
```

Comment:
While you are reviewing JSON parts ... didn't we want to add ability to just send a generic JSON Object?

Reply:
I am suspicious of "blob" objects (so not hurrying into supporting JSON objects); I would like to be able to support a `a -> HoneyObject` based on `Generic` though.

Reply:
I would leave this for another PR. The JSON changes here are to verify the structure of the JSON values passed via the Request.